README.md

```diff
@@ -182,6 +182,19 @@ merge:
 Anything that's contained between two `==COMMIT_MSG==` strings will become the
 commit message instead of whole pull request body.
 
+#### What if I don't want to put config files into each repo?
+You can enable global fallback configuration in your bulldozer config file.
+
+Global configuration will be used only when your repo config file does not exist.
+
+```yaml
+options:
+  enable_global_config: true
```

Comment:
Should we rename this to `enable_repository_default_config` ?

Reply:
I actually think we only need the `default_repository_config` field, see my other comment.